### ISSUE TITLE
[Issue]: Media data parsing issue

### ISSUE THREAD

**Issue by vevv** [MichaelYochpaz/iSubRip#42]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Linux

### Python Version

3.8.9

### Package Version

2.4.0

### Description

isubrip fails to scrape some some URLs and fails with a cryptic python error.

Changing the code to raise the error reveals that it's some issue with name parsing.
```
  File "isubrip/__main__.py", line 52, in main
    print(f"Found movie: {media_data.name} ({media_data.release_date.year})")
AttributeError: 'list' object has no attribute 'name'
```

### Output Log

```Text
isubrip https://tv.apple.com/ca/movie/umc.cmc.2jhzj3f3ic3shrf0gt5450j9z?l=en
Scraping https://tv.apple.com/ca/movie/umc.cmc.2jhzj3f3ic3shrf0gt5450j9z?l=en
Error: 'list' object has no attribute 'name'
```

**Comment by MichaelYochpaz** [comment]
Hey, thanks (again) for reporting.
I've pushed a fix, and like I mentioned on the other issue, I'll probably release a bugfix version with it tomorrow.
You can [install the package from Git](https://github.com/MichaelYochpaz/iSubRip#git-source-code) meanwhile until the release if you want to have it working.

**Comment by MichaelYochpaz** [comment]
Fixed in v2.4.1
